Sources/MapboxNavigation/EndOfRouteFeedbackCollecting.swift

```diff
@@ -0,0 +1,142 @@
+import UIKit
+import MapboxDirections
+import MapboxCoreNavigation
+import MapboxMobileEvents
+
+
+public protocol EndOfRouteFeedbackCollecting: UIViewController {
+    typealias EndOfRouteDismissalHandler = (EndOfRouteFeedback?) -> ()
+    
+//    private var endOfRouteViewController: EndOfRouteViewController // expose via protocol?
+    
+    func showEndOfRouteFeedbackCollection(destination: Waypoint?,
+                                          onDismiss: EndOfRouteDismissalHandler?) -> UIViewController
+}
+
+extension EndOfRouteFeedbackCollecting {
```

Comment:
I would like to introduce as much customization options as possible, but the majority of things are happening inside `EndOfRouteViewController`. This extension is mostly a bootstrapping code. Anyway I think I might invest more into this part to allow, for example, embedding `EndOfRouteViewController` to the top of the screen or something like that.